dash/webapp/pages/toggles.tsx

```diff
@@ -359,9 +388,35 @@ const IndexPage: FunctionComponent = () => {
                 </li>
               ))}
             </ul>
+          ) : (
+            <p>None for now, check back later…</p>
           )}
 
-          {abTests.length === 0 && <p>None for now, check back later…</p>}
+          <hr style={{ margin: '3em' }} />
+
+          <h2>Custom settings</h2>
```

Comment:
You could make it clearer that this is a "developer setting" perhaps by calling these "developer settings"? Not sure though, not blocking for merge.

Reply:
Could do!